lib/db.js

```diff
@@ -63,6 +63,10 @@ function getMongoConnectionUrl(cb) {
     return mongoConnectionStringOS2(cb);
   } else if ("openshift3" === process.env.FH_MBAAS_TYPE) {
     return mongoConnectionStringOS3(cb);
+  } else if (process.env.FH_USE_LOCAL_DB) {
+    // Backwards compatible, hard coded localhost for local development
+    process.env.FH_MONGODB_CONN_URL = 'mongodb://localhost:27017';
```

Comment:
@wei-lee I might be misunderstanding but isn't that what's done in the logic above this? I would have thought if the mongodb url was different locally the user could set the `FH_MONGODB_CONN_URL` env var? Then this logic will never get used

@david-martin Should we be using the `admin` database by default? Or would it be possible to use a named database? Like how we would usually use `FH_LOCAL`